client/components/forms/multi-checkbox/index.tsx

```diff
@@ -10,30 +10,30 @@ const noop = () => {};
 
 type OptionValue = number | string;
 
-interface Option {
-	value: OptionValue;
+interface Option< T > {
```

Comment:
For backward compatibility, it's better to set the default value to what it was before.

```suggestion
interface Option< T = OptionValue > {
```